apps/studio/styles/typography.css

```
@@ -60,6 +82,7 @@
   /* Text */
   .text-default {
     @apply text-base;
+    letter-spacing: -0.01em;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)